pimd/pim_zebra.c

```diff
@@ -1231,14 +1231,15 @@ void pim_forward_start(struct pim_ifchannel *ch)
 					__FILE__, __PRETTY_FUNCTION__,
 					source_str);
 			}
-			up->channel_oil = pim_channel_oil_add(
-				pim, &up->sg, MAXVIFS, __PRETTY_FUNCTION__);
+			pim_channel_oil_change_iif(pim, up->channel_oil,
+						   MAXVIFS,
+						   __PRETTY_FUNCTION__);
 		}
 
 		else
-			up->channel_oil = pim_channel_oil_add(
-				pim, &up->sg, input_iface_vif_index,
-				__PRETTY_FUNCTION__);
+			pim_channel_oil_change_iif(pim, up->channel_oil,
+						   MAXVIFS,
```

Comment:
why was this changed from input_iface_vif_index to MAXVIFs?

Reply:
Thanks for catching this @AnuradhaKaruppiah -> When I ported the code from internal version to external I messed this up.